docs/suspensive.org/src/pages/docs/react-query/_meta.en.json

```diff
@@ -1,6 +1,7 @@
 {
   "motivation": { "title": "Why need to use?" },
   "installation": { "title": "Installation" },
+  "command-line-interface": { "title": "Command-Line Interface" },
```

Comment:
I named the sidebar list "Command-Line Interface." If you have a better name, please let me know!